dataland-frontend/src/components/forms/parts/fields/PercentageFormField.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="form-field">
+    <UploadFormHeader :name="`${displayName} (%)`" :explanation="info" :is-required="required" />
+    <FormKit
+      type="text"
+      :name="name"
+      :validation-label="validationLabel ?? displayName"
+      :validation="`number|between:0,100|${validation}`"
+      :placeholder="placeholder ?? 'Value in %'"
+      inner-class="short"
+      v-model="percentageString"
```

Comment:
I could not find out what variable this is referring to.
I mean if I search for "percentageString" in our frontend source code, I cannot find any references.
Where does this come from?

Reply:
Removed.

Reply:
Ok. Thanks